src/AccessibilityInsights.SharedUx/Settings/ConfigurationConstants.cs

```diff
@@ -71,7 +71,7 @@ public partial class ConfigurationModel
         private const string keyTestReportPath = "TestReportPath";
         private const string keyTreeViewMode = "TreeViewMode";
         private const string keyVersion = "Version";
-        private const string keyIssueReporting= "IssueReportinGuid";
+        private const string keyIssueReporting= "IssueReportingGuid";
```

Comment:
>IssueReportingGuid [](start = 49, length = 18)

There should be a unit test that ensures that this data is correct in the JSON, and that unit test should need to change as a result of this. I see no unit test change, so it seems that perhaps the unit test was omitted originally. Can we please add it now?

Reply:
@flyingUnderTheRadar is working on the necessary unit test updates for this generally. The ConfigurationModelTests have already been broken in the feature branch--I would rather have them fixed/updated all at once than start the process in this PR.